lib/rust/ensogl/component/text/src/component/area.rs

```diff
@@ -888,32 +905,33 @@ impl AreaModel {
     fn line_truncated_with_ellipsis(
         &self,
         line: &str,
-        font_size: style::Size,
-        max_width_px: f32,
+        _font_size: style::Size,
+        _max_width_px: f32,
     ) -> String {
-        const ELLIPSIS: char = '\u{2026}';
-        let mut pen = pen::Pen::new(&self.glyph_system.borrow().font);
-        let mut truncation_point = 0.bytes();
-        let truncate = line.char_indices().any(|(i, ch)| {
-            let char_info = pen::CharInfo::new(ch, font_size.raw);
-            let pen_info = pen.advance(Some(char_info));
-            let next_width = pen_info.offset + char_info.size;
-            if next_width > max_width_px {
-                return true;
-            }
-            let width_of_ellipsis = pen::CharInfo::new(ELLIPSIS, font_size.raw).size;
-            let char_length: Bytes = ch.len_utf8().into();
-            if next_width + width_of_ellipsis <= max_width_px {
-                truncation_point = Bytes::from(i) + char_length;
-            }
-            false
-        });
-        if truncate {
-            let truncated_content = line[..truncation_point.as_usize()].to_string();
-            truncated_content + String::from(ELLIPSIS).as_str()
-        } else {
-            line.to_string()
-        }
+        // const ELLIPSIS: char = '\u{2026}';
+        // let mut pen = pen::Pen::new(&self.glyph_system.borrow().font);
+        // let mut truncation_point = 0.bytes();
+        // let truncate = line.char_indices().any(|(i, ch)| {
+        //     let char_info = pen::CharInfo::new(ch, font_size.raw);
+        //     let pen_info = pen.advance(Some(char_info));
+        //     let next_width = pen_info.offset + char_info.size;
+        //     if next_width > max_width_px {
+        //         return true;
+        //     }
+        //     let width_of_ellipsis = pen::CharInfo::new(ELLIPSIS, font_size.raw).size;
+        //     let char_length: Bytes = ch.len_utf8().into();
+        //     if next_width + width_of_ellipsis <= max_width_px {
+        //         truncation_point = Bytes::from(i) + char_length;
+        //     }
+        //     false
+        // });
+        // if truncate {
+        //     let truncated_content = line[..truncation_point.as_usize()].to_string();
+        //     truncated_content + String::from(ELLIPSIS).as_str()
+        // } else {
+        //     line.to_string()
+        // }
+        line.to_string()
```

Comment:
This should work, it's needed in Component Browser.

Reply:
As we agreed on the call, this will work again in the new upcoming PR.